test/prototype_transforms_kernel_infos.py

```diff
@@ -1184,38 +1184,38 @@ def reference_inputs_pad_bounding_box():
 def sample_inputs_perspective_image_tensor():
     for image_loader in make_image_loaders(sizes=["random"]):
         for fill in [None, 128.0, 128, [12.0], [12.0 + c for c in range(image_loader.num_channels)]]:
-            yield ArgsKwargs(image_loader, fill=fill, perspective_coeffs=_PERSPECTIVE_COEFFS[0])
+            yield ArgsKwargs(image_loader, None, None, fill=fill, coefficients=_PERSPECTIVE_COEFFS[0])
```

Comment:
Nit: While correct, can we pass keywords here to make this call more readable?

```suggestion
            yield ArgsKwargs(image_loader, startpoints=None, endpoints=None, fill=fill, coefficients=_PERSPECTIVE_COEFFS[0])
```

Reply:
Happy to make this change on the tests. I wouldn't want to make it on the main code because the arguments are positional not keyword args. Any concerns on that?

Reply:
Nono, fine with that. The test definition is just hard to read without knowledge of the exact signature of the kernel.

Reply:
Something, somewhere in the testing framework seems to send [multiple parameters](https://github.com/pytorch/vision/actions/runs/3392776645/jobs/5639353266) to the kernel. I don't want to waste too much time on this, I've reverted the named params change and we can do the optimization on a follow up as it doesn't affect the main implementation or the quality of the tests.